test/Bugs/rlexe.xml

```diff
@@ -373,4 +373,11 @@
       <files>valueInfoLossBug.js</files>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug13383062.js</files>
+      <compile-flags>-maxinterpretcount:2 -off:simplejit -off:lossyinttypespec -bgjit- -off:redeferral</compile-flags>
```

Comment:
>redeferral [](start = 92, length = 10)

this switch isn't really required for your test case. I had just used it to rule out redeferral